src/lib/divider/divider.md

```diff
@@ -0,0 +1,64 @@
+`<mat-divider>` is a container component that wraps and formats a series of line items. As the base
+list component, it provides Material Design styling, but no behavior of its own.
+
+<!-- example(divider-overview) -->
+
+
+### Simple divider
+
+An `<mat-divider>` element can be used on its own to create a vertical line styled with a Material theme
+
+```html
+<mat-divider></mat-divider>
+```
+
+Here are the available global options:
+
+| Name            | Type    | Values      | Description                               |
+| --------------- | ------- | ----------- | ----------------------------------------- |
+| inset           | boolean | true, false | Whether the divider is an inset divider   |
+| vertical        | boolean | true, false | Whether the divider is a vertical divider |
```

Comment:
IMO the values column is covered by them both having boolean type. Further, I'm not sure this table is even necessary, since it will be available on the api page.

Reply:
Originally pulled from list, but now I see it's been removed. Will fix.